test/lit.cfg.py

```diff
@@ -58,8 +58,8 @@
 tools = [
     'arcilator', 'circt-as', 'circt-capi-ir-test', 'circt-capi-om-test',
     'circt-capi-firrtl-test', 'circt-capi-firtool-test', 'circt-dis',
-    'circt-opt', 'circt-reduce', 'circt-translate', 'firtool', 'hlstool',
-    'om-linker', 'ibistool'
+    'circt-opt', 'circt-reduce', 'circt-translate', 'circt-verilog', 'firtool',
```

Comment:
:partying_face: Oooh awesome catch :100: Thanks. Yeah this would break in a build without Slang. Or I guess the tools list doesn't care much, but the list of dependencies does, and I have to skip the Verilog tests if Slang is disabled.